### PR TITLE
Merge pull request #2 from amhendrix/weatherlookup_branch

### PR DESCRIPTION
**Pull request by amhendrix**
Getting some weather data from 2 different sources

### CODE REVIEW

File: weather.py [hunk 1]
```diff
@@ -33,7 +40,7 @@
 #Using the weather api for ninjas API
 url = "https://weather-by-api-ninjas.p.rapidapi.com/v1/weather"
 
-querystring = {"city":"Seattle"}
+querystring = {"city":"Asheville","state":"NC","country":"US","units":"imperial","lang":"en","format":"json"}
```
Comment (spackle0) [inline weather.py:43]: This looks good. Where was it you were talking about the nested "if" statement?
Did you have problems above with the argparse?

Comment (amhendrix) [reply weather.py:43]: I think I was over complicating things at first. 
I was just starting to get started on argparse when I committed this.

File: weather_functions.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
+import json
+import requests
+import argparse
+from geopy.geocoders import Nominatim
+from urllib.parse import urlencode
+from datetime import datetime
+
```
Comment (amhendrix) [inline weather_functions.py:7]: Made a file for all the functions to make the code easier to read

Comment (spackle0) [reply weather_functions.py:7]: Good! Keep it modular.

**Pull request merged**